mobile/src/bundles/common/components/selector/selector.tsx

```diff
@@ -57,7 +79,9 @@ const Selector: React.FC<Properties> = ({ options }) => {
                 ]}
                 onPress={toggleIsListVisible}
             >
-                <Text category={TextCategory.LABEL}>{selectedOption}</Text>
+                <Text category={TextCategory.LABEL}>
+                    {selectedOptions.join(', ')}
+                </Text>
```

Comment:
Can you share how it looks like

Reply:
![few options](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/104503014/81b7110e-7eab-4d0d-8685-9562f6027249)
![one option](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/104503014/078bfeb5-98da-4b2c-995a-801e12f4597f)